content/applications/sales/sales/send_quotations/pdf_quote_builder.rst

```diff
@@ -0,0 +1,209 @@
+for Odoo to auto-fill the PDF content with information related to the quote from the Odoo database,
+like names, prices, etc.
+
+Dynamic text values are form components (text inputs) that can be added in a PDF file, and Odoo
+automatically fills it in with information related to the quote. Odoo fills these components
+according to their name. Doing so streamlines the writing of personalized quotations.
+
+Dynamic text values
+-------------------
+
+Below are common dynamic text values used in custom PDFs, and what they represent:
+
+- :guilabel:`name`: Sales Order Reference
+- :guilabel:`partner_id_name`: Customer Name
+- :guilabel:`user_id_name`: Salesperson Name
+- :guilabel:`amount_untaxed`: Untaxed Amount
+- :guilabel:`amount_total`: Total Amount
+- :guilabel:`delivery_date`: Delivery Date
+- :guilabel:`validity_date`: Expiration Date
+- :guilabel:`client_order_ref`: Customer Reference
+
+Product-specific dynamic text values are as follows:
+
+- :guilabel:`description`: Product Description
+- :guilabel:`quantity`: Quantity
+- :guilabel:`uom`: Unit of Measure (UoM)
+- :guilabel:`price_unit`: Price Unit
+- :guilabel:`discount`: Discount
+- :guilabel:`product_sale_price`: Product List Price
+- :guilabel:`taxes`: Taxes name joined by a comma (`,`)
+- :guilabel:`tax_excl_price`: Tax Excluded Price
+- :guilabel:`tax_incl_price`: Tax Included Price
+
+.. example::
+   A sample PDF being built using common dynamic text values (:guilabel:`name` and
+   :guilabel:`partner_id_name`). When uploaded into the database, Odoo auto-populates those fields
+   with the information from their respective fields.
+
+   In this case, Odoo would auto-populate the Sales Order Reference in the :guilabel:`name` dynamic
+   text field, and the Customer Name in the :guilabel:`partner_id_name` field.
+
+   .. image:: pdf_quote_builder/pdf-quote-builder-sample.png
+      :align: center
+      :alt: PDF quote being built using common dynamic placeholders.
+
+Once the PDF file(s) are complete, save them to the computer's hard drive, and proceed to upload
+them to Odoo via :menuselection:`Sales app --> Configuration --> Settings --> PDF Quote builder`.
+
+Upload the created PDF in the :guilabel:`Header pages` or :guilabel:`Footer pages` field.
+
+Once the upload(s) are complete, click :guilabel:`Save`.
+
+Add PDF to product
+==================
+
+In Odoo *Sales*, it's also possible to add a custom PDF to a product form. When a PDF is added to a
+product, and that product is used in a quotation, that PDF is also inserted in the final PDF.
+
+To add a custom PDF to a product, start by navigating to :menuselection:`Sales app --> Products -->
+Products`, and selected the desired product to which a custom PDF should be added.
+
+.. note::
+   A document could also be added to a product variant, instead of a product. If there are documents
+   on a product *and* on its variant, **only** the documents in the variant are shown.
```

Comment:
Can this doc include brief instructions on how to attach a document to a product *variant* as well?